header/verify.go

```diff
@@ -26,6 +26,11 @@ func (eh *ExtendedHeader) IsExpired() bool {
 	return !expirationTime.After(time.Now())
 }
 
+// IsRecent checks if header is recent against the given blockTime.
+func (eh *ExtendedHeader) IsRecent(blockTime time.Duration) bool {
+	return time.Since(eh.Time) <= blockTime // TODO @renaynay: should we allow for a 5-10 block drift here?
```

Comment:
Let's just put this in an issue

Reply:
For now, let's not allow any drift unless it becomes an issue.

Reply:
Drift, yes, but we still need to figure out if we lag on block or not